react/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "private": true,
+  "jest": {
+    "setupFilesAfterEnv": "./setupTests.js"
```

Comment:
test-tools reads setupTests automatically

Reply:
ops, nope hehe

Reply:
I read the implementation to see how it's done 😉